packages/paste-core/layout/grid/src/utils.tsx

```diff
@@ -12,34 +11,66 @@ import {
 
 type Vertical = ResponsiveValue<boolean>;
 
-// Gets the gutter and returns the value to be used as negative margin to Grid
-export const getOuterGutterPull = (theme: ThemeShape, gutter?: Space): Margin => {
-  if (Array.isArray(gutter)) {
-    return (gutter as SpaceOptions[]).map((value: SpaceOptions) => {
-      return `-${theme.space[value]}` as SpaceOptions;
-    });
+export const getCorrespondingNegativeSpacingToken = (gutter: SpaceOptions): SpaceOptions => {
+  switch (gutter) {
+    default:
+      return 'space0';
+    case 'space10':
+      return 'spaceNegative10';
+    case 'space20':
+      return 'spaceNegative20';
+    case 'space30':
+      return 'spaceNegative30';
+    case 'space40':
+      return 'spaceNegative40';
+    case 'space50':
+      return 'spaceNegative50';
+    case 'space60':
+      return 'spaceNegative60';
+    case 'space70':
+      return 'spaceNegative70';
+    case 'space80':
+      return 'spaceNegative80';
+    case 'space90':
+      return 'spaceNegative90';
+    case 'space100':
+      return 'spaceNegative100';
+    case 'space110':
+      return 'spaceNegative110';
+    case 'space120':
+      return 'spaceNegative120';
+    case 'space130':
+      return 'spaceNegative130';
+    case 'space140':
+      return 'spaceNegative140';
+    case 'space150':
+      return 'spaceNegative150';
+    case 'space160':
+      return 'spaceNegative160';
+    case 'space170':
+      return 'spaceNegative170';
+    case 'space180':
+      return 'spaceNegative180';
+    case 'space190':
+      return 'spaceNegative190';
+    case 'space200':
+      return 'spaceNegative200';
   }
-
-  if (gutter) {
-    return `-${theme.space[gutter as SpaceOptions]}` as SpaceOptions;
-  }
-
-  return 'auto';
 };
 
-// Gets the gutter and returns the value to be used as padding for Columns
-export const getColumnGutters = (theme: ThemeShape, gutter?: Space): Padding => {
+// Gets the gutter and returns the value to be used as negative margin to Grid
+export const getOuterGutterPull = (gutter?: Space): Margin => {
   if (Array.isArray(gutter)) {
     return (gutter as SpaceOptions[]).map((value: SpaceOptions) => {
-      return `${theme.space[value]}` as SpaceOptions;
+      return getCorrespondingNegativeSpacingToken(value);
```

Comment:
This could be simplified as a string replacement `value.replace('space', 'spaceNegative')` removing the need for the function entirely.

Reply:
I've modified that, and updated the tests.